Documentation/go_client.md

```diff
@@ -0,0 +1,18 @@
+# etcd go client API
+
+This document describes tips for using the etcd go client (the package `github.com/coreos/etcd/client`).
+
+
+## Endpoint Selection Mode
+
+This section describes how to configure a policy of choosing a request destination node in etcd client programs. It will affects performance (latency and throughput) of request processing in the etcd cluster.
+
+A program that uses etcd client creates its instance of etcd client object with calling `client.New()`. The function requires an instance of `client.Config` as its parameter. The `client.Config` has a member named `SelectionMode`, typed as an enum `client.EndpointSelectionMode`. With the member, the program can specify the policy of choosing a request destination node in the etcd cluster.
```

Comment:
Programs call `client.New()` to create an etcd client instance, passing configuration through the `client.Config` parameter. `client.Config`'s `SelectionMode` member is an `EndpointSelectionMode` `enum` that specifies the policy for choosing the etcd cluster node to which requests are sent.